### PR TITLE
GraphQL mutation

### PR DESCRIPTION
**Pull request by levity**

### CODE REVIEW

**Comment by levity** [comment]
No need for all of you to review this -- one's enough, whoever can get to it soonest.

**Comment by Connoropolous** [comment]
reviewing

File: api/graphql/schema.graphql [hunk 1]
```diff
@@ -49,3 +53,16 @@ type Query {
...
   community(id: ID, slug: String): Community
 }
+
+input MeInput {
+  name: String
+  email: String
+  avatarUrl: String
+  bannerUrl: String
+  bio: String
+  url: String
```
Comment (Connoropolous) [inline api/graphql/schema.graphql]: what led to the selection of these specific attributes, from the whole 'whitelist' that is under validateAndSave, and the difference between the graphql mutation and the possible mutation available via the original controller method

Comment (levity) [inline api/graphql/schema.graphql]: This was the set that I was pretty confident we would continue to directly update as simple strings. I think we might want to create separate mutations for tags, password, and social media links. And then there are a few other fields that are not used anymore.

**Comment by levity** [comment]
Here's an example of what a mutation looks like. You specify what you want to be returned just as you do for a query:

```graphql
mutation {
  updateMe(changes: {bio: "I am the very model"}) {
    id
    name
    email
    bio
    posts(first: 3) {
      title
    }
  }
}
```

The arguments can also be separated out:

```graphql
mutation ($changes: MeInput) {
  updateMe(changes: $changes) {
    id
    name
    email
    bio
    posts(first: 3) {
      title
    }
  }
}

# this part would go in the "Query Variables" section of GraphiQL

{
  "changes": {
    "bio": "I am the very model"
  }
}
```

File: api/graphql/schema.graphql (hunk 1 above)
Comment (Connoropolous) [inline api/graphql/schema.graphql]: makes sense to me 👍

**Pull request merged**